package-rfcx/rfcx/client.py

```diff
@@ -242,25 +268,48 @@ def downloadStreamSegments(self, dest_path=None, stream_id=None, min_date=None,
         if not isinstance(max_date, datetime.datetime):
             print("max_date is not type datetime")
             return
-        
+
         if dest_path == None:
             if not os.path.exists('./audios'):
                 os.makedirs('./audios')
             else:
-                print('`audios` directory is already exits. Please specific the directory to save audio path or remove `audios` directoy')
+                print(
+                    '`audios` directory is already exits. Please specific the directory to save audio path or remove `audios` directoy'
+                )
                 return
 
-        return audio.downloadStreamSegments(self.credentials.id_token, dest_path, stream_id, min_date, max_date, gain, file_ext, parallel)
-
-    def streams(self, keyword=None, limit=1000, offset=0):
+        return audio.downloadStreamSegments(self.credentials.id_token,
+                                            dest_path, stream_id, min_date,
+                                            max_date, gain, file_ext, parallel)
+
+    def streams(self,
+                organizations=None,
+                projects=None,
+                created_by=None,
+                keyword=None,
+                is_public=True,
+                is_deleted=False,
+                limit=1000,
+                offset=0):
         """Retrieve a list of streams
 
         Args:
-            keyword:(optional, default= '') Match streams with name
+            organizations: List of organization ids
+            projects: List of organization ids
+            created_by: The stream owner. Have 3 options: None, me, or collaborators
+            keyword: Match streams name with keyword
+            is_public: (optional, default= True) Match public or private streams
+            is_deleted: (optional, default= False) Match deleted streams
```

Comment:
Some of these things will change a bit in the coming weeks, so keep an eye out for any task that Stas or I change it and try to keep up-to-date on SDK.